app/account/forms.py

```diff
@@ -4,10 +4,10 @@
     StringField,
     PasswordField,
     BooleanField,
-    SubmitField
+    SubmitField,
```

Comment:
Technically no, but it's good style to leave an extra comment in situations like this

For example, Go won't even compile if you leave out the dangling comma!